### ISSUE TITLE
Error when queue retried: exponentialBackOff is not a function

### ISSUE THREAD

**Issue by de3** [arangodb/arangodb#1977]
## my environment running ArangoDB
I'm using the latest ArangoDB of the respective release series:
- [x] 3.0.3

On this operating system:
- [x] MacOS, version: 10.11.6
  - [x] docker - official docker library

### Foxx
How to reproduce:

create service for queue

```
'use strict';
const request = require('@arangodb/request');
const util = require('util');
const body = module.context.argv[0];
const config = module.context.configuration;

const opts = {
  url: config.functionUrl,
  method: 'POST',
  body: body,
  json: true
};

const res = request(opts);

if (res.body) {
  if (res.statusCode !== 200) {
    throw new Error(util.format(
      'Server returned HTTP status %s with message: %s',
      res.statusCode,
      res.body
    ));
  }
} else {
  throw new Error('Server sent empty response with status ' + res.statusCode);
}
if (res.statusCode != 200) {
  throw new Error(res.message);
}

console.log(res.body)

module.exports = res.body;
```

push job:

```
queue.push(
    {mount: '/queue', name: 'queue'},
    {asd:"asdasd"},
    {
      success: function () {
        console.log('Yay!');
      },
      failure: function () {
        console.log('Fail');
      },
      maxFailures: 2
    }
  );
```

and then, try to make response code from 3rd party is not 200.

It will return:

```
TypeError: exponentialBackOff is not a function 
at getBackOffDelay (/usr/share/arangodb3/js/server/modules/@arangodb/foxx/queues/worker.js:75:10)
at db._executeTransaction.action (/usr/share/arangodb3/js/server/modules/@arangodb/foxx/queues/worker.js:156:33)
at ArangoDatabase._executeTransaction
(/usr/share/arangodb3/js/server/modules/@arangodb/arango-database.js:119:10)
at Object.exports.work (/usr/share/arangodb3/js/server/modules/@arangodb/foxx/queues/worker.js:131:6)
at eval (eval at undefined, <anonymous>:8:59)
at eval (eval at undefined, <anonymous>:11:17)
at eval (eval at undefined, <anonymous>:11:29)
```

and when I check internal module, there are no `exponentialBackOff` function.

**Comment by dothebart** [comment]
the function was removed with a80d6635ab14aef031ba4d3739ac84eec3aedfc9 - still investigating.

**Comment by jsteemann** [comment]
Should be fixed in 2.8, 3.0 and devel branches now via the above commits.

**Comment by dothebart** [comment]
@de3 since we just released 3.0.4, you can obtain the fix by downloading this file:
https://raw.githubusercontent.com/arangodb/arangodb/3.0/js/server/modules/%40arangodb/foxx/queues/worker.js

and replace `js/server/modules/@arangodb/foxx/queues/worker.js` on your installation with it.

The next release containing this fix will be 3.0.5.
Thanks for reporting.
